src/wallet/wallet_tx_builder.cpp

```diff
@@ -177,25 +192,232 @@ SpendableInputs WalletTxBuilder::FindAllSpendableInputs(
     return wallet.FindSpendableInputs(selector, minDepth, std::nullopt);
 }
 
-InputSelectionResult WalletTxBuilder::ResolveInputsAndPayments(
-        const CWallet& wallet,
+static size_t NotOne(size_t n)
+{
+    return n == 1 ? 2 : n;
+}
+
+/// ISSUES:
+/// • z_shieldcoinbase currently checks the fee in advance of calling PrepareTransaction in order to
+///   determine the payment amount. But that fee possibly isn’t correct. We could perhaps have _no_
+///   payments in z_shieldcoinbase, but provide an explicit change address? (Don’t expose this via
+///   RPC, and give it a different name, e.g. `NetFundsRecipient` in the WalletTxBuilder interface)
```

Comment:
I don't entirely understand this. Please file an issue with more explanation.

Reply:
This isn’t an issue about anything that currently exists in master, so I’ll remove it and include it in my PR that makes the relevant change.